web/themes/custom/digital_gov/templates/page/page.html.twig

```diff
@@ -62,22 +62,4 @@
 </div>
 
 <div id="clicks_iframe" class="display-none"></div>
-
-{# NOT-MIGRATED #}
-{# The glossary will NOT be included in this way, leaving it just to remember
-  to implement it.
-<!-- Set the glossary terms as a variable -->
-<script type="text/javascript">
-    {{- $terms := .Params.glossary -}}
-    {{- if (not .IsNode) -}}
-      {{- $terms = .Parent.Params.glossary -}}
-    {{- end -}}
-    {{- $glossaryPath := (printf "%s/%s" "glossary/" $terms | absURL) -}}
-  let glossaryPath = null;
-  {{- if and ($terms) (os.FileExists (printf "%s/%s" "themes/digital.gov/static/glossary/" $terms)) -}}
-  glossaryPath = "{{ $glossaryPath | absURL }}";
-  {{- end -}}
-  // Needed to pass relURL to JS
-  const glossaryIcon = "{{ "uswds/img/sprite.svg#expand_more" | relURL }}"
-</script>
 #}
```

Comment:
Ending comment left hanging on for dear life.
Also, these two templates are pretty much identical now:
![image](https://github.com/user-attachments/assets/41e3eabd-33e6-4792-95e6-885ab670c3ac)
Are we sure we need a node specific override?

Reply:
I don't think so - I've removed the node specific page template and removed the menu vars which are null.